inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -146,8 +146,13 @@ def arxiv_plot_extract(obj, eng):
                 return
 
             lb = LiteratureBuilder(source='arxiv', record=obj.data)
+            file_extension_pattern = re.compile('.+(\.[a-zA-Z]+)$')
             for index, plot in enumerate(plots):
```

Comment:
IIRC, you don't need the `enumerate` as the `index` is not used.

Reply:
it is used in the number ;) Here https://github.com/inspirehep/inspire-next/pull/2870/files/92b21ff84972aa6f31a116c91d13263c2805f475#diff-d5b37b2f12fac54ce9b26fffdb95b1ceL154